platform/android/export/export_plugin.cpp

```diff
@@ -2342,11 +2283,11 @@ List<String> EditorExportPlatformAndroid::get_binary_extensions(const Ref<Editor
 	return list;
 }
 
-void EditorExportPlatformAndroid::_update_custom_build_project() {
+void EditorExportPlatformAndroid::_update_custom_build_project(const String &p_template_path) {
```

Comment:
These changes can be simplified if we choose to use the `user://` build template mutably.